lib/ecto/repo/schema.ex

```diff
@@ -18,6 +18,10 @@ defmodule Ecto.Repo.Schema do
     do_insert_all(repo, adapter, nil, {nil, table}, rows, opts)
   end
 
+  def insert_all(repo, adapter, table, rows, opts) when is_tuple(table) do
```

Comment:
I had thought about that, too. Maybe even checking `is_binary` to match the previous.

and thank you man! You're my hero.